### PR TITLE
refactor mpt_issuance_id into SLE::getJson

### PR DESCRIPTION
**Pull request by shawnxie999**
## High Level Overview of Change

Missing mpt_issuance_id in `ledger_entry` response if the object type is `MPTokenIssuance`. Although not necessary (since the user probably already knows the mpt id in their request), we still should make things consistent by injecting the id into the response if it's a MPTokenIssuance object.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Performance (increase or change in throughput and/or latency)
- [ ] Tests (you added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation update
- [ ] Chore (no impact to binary, e.g. `.gitignore`, formatting, dropping support for older tooling)
- [ ] Release

### API Impact

- [ ] Public API: New feature (new methods and/or new fields)
- [ ] Public API: Breaking change (in general, breaking changes should only impact the next api_version)
- [ ] `libxrpl` change (any change that may affect `libxrpl` or dependents of `libxrpl`)
- [ ] Peer protocol change (must be backward compatible or bump the peer protocol version)

### CODE REVIEW

File: src/xrpld/rpc/handlers/LedgerEntry.cpp [hunk 1]
```diff
@@ -774,7 +774,12 @@ doLedgerEntry(RPC::JsonContext& context)
         }
         else
         {
-            jvResult[jss::node] = sleNode->getJson(JsonOptions::none);
+            auto sleJson = sleNode->getJson(JsonOptions::none);
```
Comment (gregtatcam) [inline src/xrpld/rpc/handlers/LedgerEntry.cpp]: Somehow this doesn't feel right. We did specialized parsing above and now we do it again.

Comment (shawnxie999) [reply src/xrpld/rpc/handlers/LedgerEntry.cpp]: The specialized parsing above is only to find which object index given the request params. Here we are injecting the synthetic field after we've retrieved the object from DB. So I do believe this is the right place.

Comment (shawnxie999) [reply src/xrpld/rpc/handlers/LedgerEntry.cpp]: But, I think we can refactor this. I think i can move this code to STLedgerEntry, let me refactor it

Comment (gregtatcam) [reply src/xrpld/rpc/handlers/LedgerEntry.cpp]: Sure. But we have already parsed all ledger entries and we are of doing it again. What if more ledger entries require this special processing then we pretty much have to repeat specialized type parsing again. Maybe we could have a global json variable, which is set during the ledger entry parsing and then the values from this variable are copied into sleJson?

**Pull request merged**